src/tye/StandardOptions.cs

```diff
@@ -81,6 +81,17 @@ public static Option Debug
             }
         }
 
+        public static Option IncludeLatestTag
+        {
+            get
+            {
+                return new Option(new[] { "--include-latest-tag", }, "Push a docker image with 'latest' tag as well as versioned tag")
```

Comment:
When all would we want to do this latest tag?

Reply:
You see use of the `latest` tag in a lot of examples and documentation. Not meant to be used by the container directly necessarily, but more like a fallback container that represents the latest stable image.

For example, they reference it in the "Example" section of the best practice image tagging docs that we have referenced a couple of times https://learn.microsoft.com/en-us/azure/container-registry/container-registry-image-tag-version#example

Looking back at it this morning though, I think we should have a separate latest per environment (ex. imageName:prod-latest) instead of just a general `latest`.  Going to quick make that change in here.

Reply:
Appears the main use of a stable tag would be for updating a base image (e.g., some security update or something in a base image perhaps).  I'm sure there is a reason, just trying to understand the full use case and purpose.

> Recommendation: Use stable tags to maintain base images for your container builds. Avoid deployments with stable tags, because those tags continue to receive updates and can introduce inconsistencies in production environments.
> 
> Stable tags mean a developer, or a build system, can continue to pull a specific tag, which continues to get updates. Stable doesn’t mean the contents are frozen. Rather, stable implies the image should be stable for the intent of that version. To stay “stable”, it might be serviced to apply security patches or framework updates.
>

Reply:
Doing a bit more reading on it here this morning. It seems like it's just a tag to have out there for quick and dirty reference, but not to be used by any production services. I still don't have a full plan for our image tagging plan (hoping to have a conversation and talk through it at some point later today), but I think it's nice to have the latest one out there for our most recent build. 

May end up switching the default to false and not using it, but good to have the ability to include the latest tag for now.

Reply:
👍